web_src/js/features/repo-issue.ts

```diff
@@ -671,11 +671,13 @@ export async function initSingleCommentEditor($commentForm) {
   const commentButton = document.querySelector('#comment-button');
   opts.onContentChanged = (editor) => {
     const editorText = editor.value().trim();
+    const isUploading = editor.attachedDropzoneInst.getUploadingFiles().length != 0;
```

Comment:
Actually `attachedDropzoneInst` was not designed to be accessed out of the editor objection, it would cause unnecessary dependencies and hard to refactor in the future.

<details>

![image](https://github.com/user-attachments/assets/c184c1b8-d79d-4d46-9fde-92a98d409d62)

</details>

Actually, this quick fix only works partially but it is not a complete solution. You could see that the "close" button is still enabled and you could click it to submit the form and close the issue.

Reply:
I forgot to disable the status button while uploading, I fixed it following your advice. Thank you!